docs/typescript.mdx

```diff
@@ -286,7 +286,8 @@ declare module '@emotion/react' {
 
 // You are also able to use a 3rd party theme this way:
 import '@emotion/react'
-import { MuiTheme } from 'material-ui'
+// Import of the Material-UI theme might change depending on the version you are using 
+import { MuiTheme } from 'material-ui' 
```

Comment:
well, this was just an example - maybe it would be beneficial to just change this to smth like:
```suggestion
import { LibTheme } from 'some-lib' 
```

WDYT?

Reply:
Sounds better!